satpy/utils.py

```diff
@@ -193,7 +207,21 @@ def lonlat2xyz(lon, lat):
 
 
 def xyz2lonlat(x, y, z, asin=False):
-    """Convert cartesian to lon lat."""
+    """Convert cartesian to lon lat.
+
+    For a sphere with unit radius, convert cartesian coordinates to spherical
+    coordinates longitude and latitude.
+
+    Args:
+        x (number or array of numbers): x-coordinate, unitless
+        y (number or array of numbers): y-coordinate, unitless
+        z (number or array of numbers): z-coordinate, unitless
+        asin (optional, bool): If true, use arcsin for calculations.
+            If false, use arctan2 for calculations.
+
+    Returns:
+        (lon, lat): Longitude and latitude in °.
```

Comment:
It isn't a huge deal, but in the future I think I'd rather have this read "degrees". It is less "jarring" and probably handled better by screen readers.

Reply:
Fixed in 7d8ff7b2.